FFMQRLib/Offsets.cs

```diff
@@ -36,18 +36,24 @@ public class RomOffsets
 		public const int CreditsLengthReadTo = 0x9AD2; // Bank 00
 		public const int CreditsLengthTheEnd = 0x80B8; // Bank 03
 
-		// Enemies' Attacks
-		public const int EnemiesAttacksAddress = 0xBC78; // Bank 02
-		public const int EnemiesAttacksBank = 0x02;
-		public const int EnemiesAttacksQty = 0xA9;
-		public const int EnemiesAttacksLength = 0x07;
+		// Attacks
+		public const int AttacksAddress = 0xBC78; // Bank 02
+		public const int AttacksBank = 0x02;
+		public const int AttacksQty = 0xA9;
+		public const int AttacksLength = 0x07;
 
 		// Enemies' Stats
 		public const int EnemiesStatsAddress = 0xC275; // Bank 02
 		public const int EnemiesStatsBank = 0x02;
 		public const int EnemiesStatsQty = 0x53;
 		public const int EnemiesStatsLength = 0x0e;
 
+		// Enemies' Attacks
```

Comment:
move this info into the class for easier reference (again, in insight, this info shouldn't be referenced by anything else, there's no need to expose it globally)

Reply:
I copied the same coding style as used in other tables. If we want to change this, I would prefer doing this in a separate PR that does it project-wide.